pkg/service/credential/model.go

```diff
@@ -60,16 +61,19 @@ type GetCredentialStatusRequest struct {
 }
 
 type GetCredentialStatusResponse struct {
```

Comment:
is there a way to enforce mutual exclusion on these statuses? perhaps a validate method on each of them?

Reply:
Yea that is better:

Added:
```
func (csr CreateCredentialRequest) isStatusValid() bool {
	if csr.Revocable && csr.Suspendable {
		return false
	}
	return true
}
```

Reply:
`return !(csr.Revocable && csr.Suspendable)`